liquidpromptrc-dist

```diff
@@ -178,4 +178,12 @@ LP_ENABLE_ERROR=1
 # Ex: ("/root" "/home/me/large-remove-svn-repo")
 LP_DISABLED_VCS_PATHS=()
 
+# Show current Kubernetes kubectl context
+LP_ENABLE_KUBECONTEXT=0
+
+# Delimiter to shorten kubectl context.
+# E.g. when your context names are dev-cluster and test-cluster, set to "-"
+# in order to output "dev" and "test" in prompt.
+LP_DELIMITER_KUBECONTEXT=
+
```

Comment:
I just added a conflict here with my latest merge, sorry about that. You'll need to rebase and fix that conflict.

No rush, if anything else comes up before the v2.0.0 release, it could add more conflicts too. I doubt it though, still no bug reports on v2.0.0-rc.1

Reply:
v2.0.2 is released, so now is a good time to rebase and fix the few issues. Then this is ready to merge.

Reply:
Thanks for the reminder. I did not get around to it in the last few weeks :(